elasticassignment/ageofempires/management/commands/command_classes.py

```diff
@@ -0,0 +1,206 @@
+from abc import abstractmethod
+from elasticsearch.helpers import bulk
+import requests
+import elasticsearch
+from django.conf import settings
+
+class AOElasticsearch:
+
+    def make_connection():
+        return elasticsearch.Elasticsearch([settings.CONFIG])
+
+    def getAPIdata(url:str):
+        response_data = requests.get(url)
+        return response_data.json()
+
+    @abstractmethod
+    def make_index():
+        pass
+
+    @abstractmethod
+    def populate_index(url:str):
+        pass
+
+
+class Civilizations(AOElasticsearch):
+
+    es = AOElasticsearch.make_connection()
+
+    def make_index():
+        request_body = {
+            "settings" : {
+                "index":{
+                    "number_of_shards": settings.NUMBER_OF_SHARDS,
+                    "number_of_replicas": settings.NUMBER_OF_REPLICAS
+                }
+            },
```

Comment:
please create a `__get_settings` method.
and also a `__get_mappings` method.

We'll further refactor this part in next review cycle.